libs/platform/src/lib/thumbnail/thumbnail-image/thumbnail-image.component.html

```diff
@@ -5,9 +5,8 @@
         [class]="isHorizontal ? 'fdp-thumbnail-image--horizontal' : 'fdp-thumbnail-image'"
         [class.fdp-thumbnail-image--selected]="image.selected"
         [class.fdp-thumbnail-image--overlay]="image.overlayRequired"
-        (click)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick(image)"
-        (keydown.space)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick(image)"
-        tabindex="0"
+        (click)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick($event, image)"
+        (keydown.space)="image.overlayRequired ? openDialog(image, mediaList) : thumbnailClick($event, image)"
```

Comment:
maybe it would be better to have one method, that is used in both handlers since they're the same?